libcodechecker/analyze/analyzers/analyzer_clang_tidy.py

```diff
@@ -116,3 +117,41 @@ def construct_analyzer_cmd(self, res_handler):
         except Exception as ex:
             LOG.error(ex)
             return []
+
+    @classmethod
+    def resolve_missing_binary(cls, configured_binary, env):
```

Comment:
Though a utility could work. But not the base class thing. I'll do it into `util`.